src/dvclive/studio.py

```diff
@@ -34,9 +34,8 @@ def _cast_to_numbers(datapoints):
 
 
 def _adapt_path(live, name):
-    if live._dvc_repo is not None:
-        name = rel_path(name, live._dvc_repo.root_dir)
-    return name
+    dvc_root_path = live._dvc_repo.root_dir if live._dvc_repo else None
+    return rel_path(name, dvc_root_path)
```

Comment:
Need to adapt path even when no repo exists

Reply:
minor: since we are touching this - can we add types?

Reply:
what is the semantics of a rel_path with None passed to it? why do we need to adapt it anyways if there is no repo?

Reply:
> minor: since we are touching this - can we add types?

Added types

> what is the semantics of a rel_path with None passed to it? why do we need to adapt it anyways if there is no repo?

It is only about conversion to windows-style paths and only impacts studio tests, so I refactored to leave `rel_path` untouched and include the windows formatting directly in `_adapt_path`.